subprojects/common/files/src/main/kotlin/com/avito/utils/Path.kt

```diff
@@ -0,0 +1,17 @@
+package com.avito.utils
+
+import com.avito.android.Result
+import java.nio.file.Path
+
+/**
+ * WARNING: only android 26+ supported
+ */
+@Suppress("NewApi")
```

Comment:
Suppressing here hides the restrictions.
If we consume this module in an Android library, we wouldn't see it.
Let's use `@RequiresApi`.

Reply:
can't without adding android annotations; do you want me to add it?

Reply:
Yes, why not?
We don't forbid the usage of the common libraries in Android libraries.

Reply:
yep, thanks